### DISCUSSION TITLE
How do I achieve multiple text color like this?

### DISCUSSION THREAD

**Discussion by 96312** [tailwindlabs/tailwindcss#11370] [Help] [answered]
![image](https://github.com/tailwindlabs/tailwindcss/assets/34262180/db96fcf9-3558-430d-be63-1ba5dc087f6c)

i.e without changing the `<p>` and adding a new classname?

**Comment by wongjn** [comment, marked as answer]
You could consider surrounding the colored text pieces in `<span>` elements and then style the `<span>` elements accordingly: https://play.tailwindcss.com/GzIdqrn0X6

**Reply by 96312** [reply]
top stuff mate, thanks.
